Tests/L0/ServiceFabricUpdateAppVersions/SimpleSuffix.ps1

```diff
@@ -3,46 +3,60 @@ param()
 
 . $PSScriptRoot\..\..\lib\Initialize-Test.ps1
 
+$taskPath = "$PSScriptRoot\..\..\..\Tasks\ServiceFabricUpdateAppVersions"
+
+Microsoft.PowerShell.Core\Import-Module $taskPath\Assert-SingleItem.psm1
+Microsoft.PowerShell.Core\Import-Module $taskPath\Update-ServiceVersions.psm1
+Microsoft.PowerShell.Core\Import-Module $taskPath\Update-PackageVersion.psm1
```

Comment:
[Initialize-Test.ps1](https://github.com/Microsoft/vsts-tasks/blob/5c7baeeef193d01592c39b53d17e94107c21ce60/Tests/lib/Initialize-Test.ps1) registers a mock for 'Import-Module', so modules aren't actually imported in unit tests.

I believe this behavior was done to make mocking more fine-tuned, but I'm not sure. In this case it ensures *only* these three modules are used and thus the test should fail if an unexpected module is used.